### DISCUSSION TITLE
How to view Olimar's antenna and uncracked helmet in SSBH?

### DISCUSSION THREAD

**Discussion by Jetsplit** [ScanMountGoat/ssbh_editor#157] [Q&A] [answered]
Hello!

How can I see Olimar's glowing antenna and undamaged helmet in SSBH Editor? Can I?

Basically just hoping to get it looking like how he looks in game. By default the antenna glow and the helmet are missing, even when playing his win animations etc. I can view the cracked helmets by toggling the visibility on their meshes.

Thanks in advance for the help!

**Comment by ScanMountGoat** [comment, marked as answer]
I'm pretty sure the glowing red light is an effect. SSBH Editor doesn't support effects. For hiding the cracked helmets, you can hide the meshes in the viewport by toggling off the items in the mesh tab on the right or play an animation that hides those meshes. The uncracked helmet not rendering at all is a bug and will be fixed in the next release.

**Reply by Jetsplit** [reply]
Hey @ScanMountGoat, thank you for the quick reply! That makes sense, I thought the antenna glow might have been an effects thing and that's fine that it won't display in SSBH. Great news that I wasn't doing anything wrong and that there's a planned bug fix for the uncracked helmet not showing up!

Thank you for the phenomenal tool and for answering my question!
